build-env/docker/docker-ubuntu-18-bionic/Dockerfile.build

```diff
@@ -27,6 +30,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		wget \
 		git
 
+#install tzdata package
```

Comment:
I didn't see this added to the other Dockerfiles.  Why do we need this?

Reply:
The other OS's already had this installed by default.

Reply:
ok cool.  is it required for the build?  does the docker-based kubelet build require local timezone info?  and if so, why are we forcing America/Chicago?  What effect would that have for builders in other time zones?